### PR TITLE
Complete IDP authenticator listing.

### PR DESCRIPTION
**Pull request by vihanga-liyanage**
Peek 2020-04-21 10-23

Refers: https://github.com/wso2/identity-apps/issues/354

### CODE REVIEW

File: apps/admin-portal/src/configs/ui.ts [hunk 1]
```diff
@@ -246,7 +246,14 @@ export const IdPIcons = {
     google: GoogleIdPIcon,
     twitter: TwitterIdPIcon,
     saml: SamlLogo,
-    oidc: OIDCLogo
+    oidc: OIDCLogo,
+    wsFed: WSFedLogo,
+    smsOTP: SMSOTPIcon,
+    emailOTP: EmailOTPIcon,
+	microsoft: GoogleIdPIcon,
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/configs/ui.ts:253]:
```suggestion
microsoft: GoogleIdPIcon,
```

Comment (NipuniBhagya) [inline apps/admin-portal/src/configs/ui.ts:253]:
```suggestion
microsoft: GoogleIdPIcon,
```

File: apps/admin-portal/src/configs/ui.ts [hunk 2]
```diff
@@ -246,7 +246,14 @@ export const IdPIcons = {
     google: GoogleIdPIcon,
     twitter: TwitterIdPIcon,
     saml: SamlLogo,
-    oidc: OIDCLogo
+    oidc: OIDCLogo,
+    wsFed: WSFedLogo,
```
Comment (NipuniBhagya) [inline apps/admin-portal/src/configs/ui.ts:250]:
```suggestion
wsFed: WSFedLogo,
```

File: apps/admin-portal/src/configs/ui.ts (hunk 1 above)
Comment (NipuniBhagya) [inline apps/admin-portal/src/configs/ui.ts:253]:
```suggestion
    microsoft: GoogleIdPIcon,
```

File: apps/admin-portal/src/components/identity-providers/utils/authenticator-utils.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import {
```
Comment (brionmario) [inline apps/admin-portal/src/components/identity-providers/utils/authenticator-utils.ts:1]: Shall we move the imports bellow the copyright header?

**Pull request merged**